### PR TITLE
Addresses #1133 Adds website field for projects also adds boolean to

### PR DESCRIPTION
**Pull request by jderr-mx**
model for shouldLinkExternally

# What's in this PR?

List the changes you made and your reasons for them.

Make sure any changes to code include changes to documentation.

## References

Addresses #1133 Adds `website` field to `project` model and form. Also adds `shouldLinkExternally` boolean to `project` model

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
   "scripts": {
     "build": "ember build",
     "start": "ember server",
-    "test": "ember try:one default"
+    "test": "ember try:one default",
+    "ember-exam": "ember exam --split=3 --parallel --weighted"
```
Comment (JoshSmith) [inline package.json]: Why was this added?

File: mirage/scenarios/default.js [hunk 2]
```diff
@@ -185,7 +185,8 @@ export default function(server) {
     organization,
     owner,
     slug: 'code-corps',
-    title: 'Code Corps'
+    title: 'Code Corps',
+    website: 'www.codecorps.org'
```
Comment (JoshSmith) [inline mirage/scenarios/default.js]: I think we want this to be prefixed `https://` here, since that's what the server would do.

File: package.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "build": "ember build",
     "start": "ember server",
-    "test": "ember try:one default"
+    "test": "ember try:one default",
```
Comment (JoshSmith) [inline package.json]: Would remove the trailing comma here.

**Comment by JoshSmith** [comment]
@jderr-mx it looks like the build is failing to start – not sure if it's because of that trailing comma issue or what.

File: mirage/scenarios/default.js (hunk 2 above)
Comment (jderr-mx) [inline mirage/scenarios/default.js]: Yes! I did this with the other tests, so it makes sense

**Comment by jderr-mx** [comment]
@JoshSmith I think this one is set now

**Comment by JoshSmith** [comment]
🙌
